packages/core/template/project/userGuide/FAQ.md

```diff
@@ -0,0 +1,29 @@
+---
+  layout: default.md
+  title: "FAQ"
+  pageNav: 3
+---
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+<!-- -------------------------------------------------------------------------------------------------------------------- -->
+
+# FAQ
+
+<box type="tip">
+This section answers common questions about the app or specific features.
+</box>
+
+
+
+
+<panel header="**Question:** How do I learn to use MarkBind?">
+
+**Answer:** Refer to the [User Guide](https://markbind.org/userGuide/gettingStarted.html).
```

Comment:
Can you make this refer to the actual user guide for this documentation instead? ie it shouldn't be about markbind but about the current project that this documentation is for